app/views/course/assessment/assessments/edit.json.jbuilder

```diff
@@ -27,6 +27,7 @@ end
 
 json.mode_switching @assessment.allow_mode_switching?
 json.gamified current_course.gamified?
+json.isCourseCodaveriEnabled current_course.component_enabled?(Course::CodaveriComponent)
```

Comment:
snake_case instead of camelCase in backend please

Reply:
this is going to be sent to Frontend.. if we keep this snake_case, then when we access this field, we need to use snake case

Though, I don't know why special for this `AssessmentEdit` we send snake case to frontend.. Other `json.jbuilder` sends camelCase to FE

Reply:
Yes you're right looks like majority is camelCase, leave it be then